### ISSUE TITLE
Make fails in filetime v0.1.5

### ISSUE THREAD

**Issue by dannluciano** [uutils/coreutils#682]
```
$ rustc --version
rustc 1.0.0 (a59de37e9 2015-05-13) (built 2015-05-14)

$ cargo --version
cargo 0.2.0-nightly (efb482d 2015-04-30) (built 2015-04-29)

$ make 
...
cd ~/code/rust/coreutils/deps && cargo build --package filetime --release
   Compiling filetime v0.1.5
~/.cargo/registry/src/github.com-1ecc6299db9ec823/filetime-0.1.5/src/lib.rs:130:29: 130:57 error: unresolved import `std::os::macos::fs::MetadataExt`. Could not find `macos` in `std::os`
~/.cargo/registry/src/github.com-1ecc6299db9ec823/filetime-0.1.5/src/lib.rs:130                         use std::os::$i::fs::MetadataExt;
                                                                                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
~/.cargo/registry/src/github.com-1ecc6299db9ec823/filetime-0.1.5/src/lib.rs:124:9: 143:10 note: in expansion of birthtim!
~/.cargo/registry/src/github.com-1ecc6299db9ec823/filetime-0.1.5/src/lib.rs:145:9: 151:10 note: expansion site
error: aborting due to previous error
Could not compile `filetime`.

To learn more, run the command again with --verbose.
make: *** [DEP_filetime] Error 101
```

**Comment by jbcrail** [comment]
Thanks for including the versions of Rust and Cargo. For coreutils, we are only targeting Rust stable (currently 1.2), beta, and nightly. Rust 1.0 had numerous issues that prevented coreutils from cleanly building.

I looked into https://github.com/alexcrichton/filetime and noticed that it depends on the `sys::fs::MetadataExt` module, which was not available for Mac OS until Rust 1.1.

You will need to upgrade to at least Rust 1.2 to build coreutils.

**Comment by dannluciano** [comment]
Thanks, in the README I read this: "Contributions are very welcome, and should target Rust's master branch until Rust 1.0 is released.". As version 1.0 has been released I thought it was the standard.
